code/modules/events/nothing_happens.dm

```diff
@@ -0,0 +1,12 @@
+
+// Due to our blatant lack of events, there is no real better solution that isn't worse shitcode.
```

Comment:
(This is entirely optional, but would be very nice)

You could, on Initialize() of this event, subtract the weight of every event from the base weight of 150, so nobody needs to update it ever, until we have enough events to remove this entirely